<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="searchLine">
     <property name="placeholderText">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="replaceLine">
     <property name="placeholderText">
      <string>Replace</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="text">
        <string>Replace</string>
       </property>
       <property name="icon">
        <iconset theme="edit-find-replace"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exchangeButton">
       <property name="text">
        <string>Exchange</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
